src/closedWindows/closedCard.less

```diff
@@ -1,22 +1,25 @@
 .closed-time {
-  font-size: 12px;
-  font-weight: @font-thin;
-  line-height: 1.4;
+  font-size: 10px;
+  font-weight: @font-light;
 }
 
 .closed-stock {
-  font-size: 13px;
+  font-size: 12px;
+  font-weight: 400;
 }
 
 .closed-card {
-  height: 60px;
-  border-bottom: 1px solid @separator-colour;
+  height: 70px;
```

Comment:
Yes I was thinking that the window would auto-resize and we just hadn't done that yet, but I notice before it allowed 5 so I'll push a commit to keep that with the new sizes.